main/ffnvcodec-headers/template.py

```diff
@@ -0,0 +1,15 @@
+pkgname = "ffnvcodec-headers"
+pkgver = "12.2.72.0"
+pkgrel = 0
+build_style = "makefile"
+hostmakedepends = ["pkgconf"]
+pkgdesc = (
```

Comment:
too long, something like "Headers for Nvidia codec APIs" says basically the same thing